rpcs3/Emu/SysCalls/Modules/cellSysutil.cpp

```diff
@@ -677,8 +690,8 @@ int cellSysCacheMount(vm::ptr<CellSysCacheParam> param)
 
 	//TODO: implement
 	char id[CELL_SYSCACHE_ID_SIZE];
-	strncpy(id, param->cacheId, CELL_SYSCACHE_ID_SIZE);
-	strncpy(param->getCachePath, ("/dev_hdd1/cache/" + std::string(id) + "/").c_str(), CELL_SYSCACHE_PATH_MAX);
+	strncpy_s(id, param->cacheId, CELL_SYSCACHE_ID_SIZE);
```

Comment:
@paulsapps I don't know if there is any limitations of those ID in the real consoles. however using std::string should be better - strings in C is quite a problem.